### DISCUSSION TITLE
Projects getting sponsorships

### DISCUSSION THREAD

**Discussion by sdelamo** [commonhaus/foundation#167] [Q&A]
How does it work for a project member of Commonhaus Foundation to get sponsorships? For example, GitHub Sponsors.

Is it necessary for the GitHub sponsor of the project to be directly associated with the Commonhaus legal entity and Stripe account? 

Will project sponsorships go to a common bucket shared between every Commonhaus project member, or will those sponsorship funds be allocated to the project so that it can use them, for example, for advocacy initiatives or infrastructure costs? Will Commonhaus keep a percentage of project sponsorships to cover the costs of the Commonhaus Foundation?

**Comment by ebullient** [comment]
> How does it work for a project member of Commonhaus Foundation to get sponsorships? For example, GitHub Sponsors.

We allow contributions via GitHub Sponsors and we act as a Fiscal Host on OpenCollective.

Funds can be allocated to a specific project (a restricted fund). I just realized I have some questions to follow up on for GitHub Sponsors, but projects will have separate pages on OpenCollective that allow direct contribution (JReleaser is already set up this way).

> Is it necessary for the GitHub sponsor of the project to be directly associated with the Commonhaus legal entity and Stripe account?

The Projects will be directly set up as sub-entities of Commonhaus (as restricted funds, essentially). Commonhaus would receive (and distribute) funds on their behalf.

> Will project sponsorships go to a common bucket shared between every Commonhaus project member, or will those sponsorship funds be allocated to the project so that it can use them, for example, for advocacy initiatives or infrastructure costs? 

Yes? I think we will have a mix of both. Membership dues and some other donations go to Commonhaus itself, and I would like to use these for shared/common efforts. There will also be donations specifically for projects, which would be restricted funds specifically for use by the project.

> Will Commonhaus keep a percentage of project sponsorships to cover the costs of the Commonhaus Foundation?

Open Source Collective and other similar fiscal hosts take 10% of incoming funds to cover costs. We don't have a hosting fee at this time for two reasons: we don't have staff whose salaries we have to pay, and we are shaping this as a membership organization with dues to cover what hosting fees otherwise might (given we have no salaried staff).

Depending on how things go (and grow), we may have to change that (and have the right to do so, though we would make that decision transparently).
